torchvision/models/detection/roi_heads.py

```diff
@@ -477,8 +477,12 @@ def postprocess_detections(self, class_logits, box_regression, proposals, image_
         pred_scores = F.softmax(class_logits, -1)
 
         # split boxes and scores per image
-        pred_boxes = pred_boxes.split(boxes_per_image, 0)
-        pred_scores = pred_scores.split(boxes_per_image, 0)
+        if len(boxes_per_image) == 1):
```

Comment:
Can you add a `TODO: remove this when ONNX support dynamic split sizes` so that we remember to remove this workaround in the future?